framework/Cargo.toml

```diff
@@ -55,6 +56,9 @@ clap = { version = "4.0.32", features = ["derive"] }
 semver = "1.0"
 cw-orch = { version = "0.27.0" }
 cw-orch-interchain = { version = "0.8.1" }
+cw-orch-clone-testing = { version = "0.9.0" }
```

Comment:
I assume clone test in framework adds many dependencies in this workspace, are we fine with it?

Reply:
Yes, that's also an issue. Do you think feature flagging those tests would be a good idea ?